plugins/eos/gs-plugin-eos.c

```diff
@@ -1374,3 +1374,56 @@ gs_plugin_add_updates (GsPlugin *plugin,
 	 * this will still work correctly */
 	return add_updates (plugin, list, cancellable, error);
 }
+
+gboolean
+gs_plugin_add_popular (GsPlugin *plugin,
+		       GsAppList *list,
+		       GCancellable *cancellable,
+		       GError **error)
+{
+	g_autoptr(AsProfileTask) ptask = NULL;
+	GsAppList *new_list = NULL;
+	const gchar *popular_apps[] = {"com.google.Chrome.desktop",
+				       "com.spotify.Client.desktop",
+				       "com.transmissionbt.Transmission.desktop",
+				       "com.valvesoftware.Steam.desktop",
+				       "libreoffice-calc.desktop",
+				       "libreoffice-impress.desktop",
+				       "libreoffice-writer.desktop",
+				       "net.minetest.Minetest.desktop",
+				       "net.sourceforge.Audacity.desktop",
+				       "org.debian.alioth.tux4kids.Tuxmath.desktop",
+				       "org.gnome.GIMP.desktop",
+				       "org.inkscape.Inkscape.desktop",
+				       "org.mozilla.Firefox.desktop",
+				       "org.pitivi.Pitivi.desktop",
```

Comment:
I think we should not feature pitivi until we are configured to use the upstream version, as our version is a bit broken right now.